action.yml

```diff
@@ -0,0 +1,35 @@
+
+# action.yml
+name: "Lazy Action Nodejs"
+description: "Github action to perform build, test , scan and generate image."
+inputs:
+  dockerfile_dir_path:
+    description: "Directory path to the dockerfile"
+    required: true
+  npm_test_placeholder:
+    description: "npm test run placeholder"
+    required: false
+    default: "test"
+  sonar_scan_in_docker:
+    description: "Is sonar scan running as part of Dockerfile"
+    required: false
+    default: "false"
+  sonar_scan_in_docker_target:
+    description: "sonar scan in docker target"
+    required: false
+    default: "sonarscan-env"
+  github_token:
+    description: "Github token"
+    required: true
+  container_push_enabled:
+    description: "Enable Build and Push Container Image"
+    required: false
+    default: "true"
+  ecr_repository:
+    description: "Ecr repository name"
+    required: true
+runs:
+  using: "docker"
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/nodejs-action:latest
```

Comment:
Change image tag

Reply:
I am testing pointing to this PR .I will change while merging PR to master.